app/views/middleware_topology/show.html.haml

```diff
@@ -0,0 +1,32 @@
+%div{'data-ng-app' => "mwTopologyApp", 'ng-controller' => "middlewareTopologyController"}
+  = stylesheet_link_tag 'container_topology'
+
+  .topology_header
+    .refresh
+      %label.checkbox-inline
+        %input#box{'ng-change' => "show_hide_names(checkboxModel.value)",  'ng-model' => "checkboxModel.value", :type => "checkbox", 'ng-true-value' => 'true', 'ng-false-value' => 'false'}
+        Display Names
+      %i{'ng-click' => "refresh()", :class => "btn btn-default fa fa-refresh refresh-topology"}
+
+    .legend
+      %label#selected
+      %div{'ng-if' => "kinds"}
+        %kubernetes-topology-icon{:kind => "MiddlewareServer", "tooltip-placement" => "bottom", "tooltip" => "{{legendTooltip}}"}
+          %svg.kube-topology
+            %g.MiddlewareServer{:transform => "translate(21, 21)"}
+              %circle{:r => "17"}
+              %image{:height => "20", :width => "20", :x => "-9", "xlink:href" => "/assets/100/middleware_server.png", :y => "-9"}
+          %label Middleware Servers
+        %kubernetes-topology-icon{:kind => "MiddlewareDeployment", "tooltip-placement" => "bottom", "tooltip" => "{{legendTooltip}}"}
+          %svg.kube-topology
+            %g.MiddlewareDeployment{:transform => "translate(21, 21)"}
+              %circle{:r => "17"}
+              %image{:height => "18", :width => "18", :x => "-9", "xlink:href" => "/assets/100/middleware_deployment.png", :y => "-9"}
+          %label Middleware Deployments
+
+  .alert.alert-info.alert-dismissable
+    %button.close{"aria-hidden" => "true", "data-dismiss" => "alert", :type => "button"}
+      %span.pficon.pficon-close
+    %span.pficon.pficon-info
+    %strong  Click on the legend to show/hide entities, and double click on the entities in the graph to navigate to their summary pages.
+  %kubernetes-topology-graph{:items => "items", :relations => "relations", :kinds => "kinds"}
```

Comment:
@mzazrivec : do we support the html tag translation here the same way we do in SSUI? @himdel : I remember we discussed that but do not remember the conclusion.

For javascript it is clear: use double underscore __()

But not sure if we added the support for HTML translations. If no, then this would need to be turned into .haml